language/de/acp/board.php

```diff
@@ -348,11 +348,15 @@
 
 // Cookie Settings
 $lang = array_merge($lang, array(
-	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'Hier legst du die Einstellungen fest, die verwendet werden, um Cookies an die Browser deiner Benutzer zu senden. In den meisten Fällen sollten die Standardwerte ausreichend sein. Führe Änderungen mit Bedacht durch, fehlerhafte Einstellungen könnten deine Benutzer daran hindern, sich anzumelden.',
+	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'Hier legst du die Einstellungen fest, die verwendet werden, um Cookies an die Browser deiner Benutzer zu senden. In den meisten Fällen sollten die Standardwerte ausreichend sein. Führe Änderungen mit Bedacht durch, fehlerhafte Einstellungen könnten deine Benutzer daran hindern, sich anzumelden. Wenn du Probleme mit Benutzern hast, die nicht angemeldet bleiben, besuche die <b><a href="https://www.phpbb.com/support/go/cookie-settings/">phpBB.com Knowledge Base - Fixing incorrect cookie settings</a></b> (<a href="https://www.phpbb.de/go/cookie-settings/" rel="external">Deutschsprachiger Artikel</a>).',
 
 	'COOKIE_DOMAIN'				=> 'Cookie-Domain',
+	'COOKIE_DOMAIN_EXPLAIN'		=> 'In den meisten Fällen ist die Cookie-Domain optional. Lasse diese leer, wenn du dir unsicher bist.<br /><br /> Falls du dein Board mit anderer Software integriert hast oder mehrere Domains verwendest, dann kannst du die benötigte Cookie-Domain wie folgt bestimmen: Falls du Domainkombinationen wie <i>example.com</i> und <i>forums.example.com</i> oder auch <i>forums.example.com</i> und <i>blog.example.com</i> verwendest, dann entferne die Subdomains bis du den gemeinsamen Domainteil gefunden hast, in diesem Beispiel <i>example.com</i>. Nun füge einen Punkt vor den gemeinsamen Domain-Namen und füge diesen in das Feld ein. In unserem Beispiel würdest du also <i>.example.com</i> einfügen (beachte den Punkt am Anfang).',
```

Comment:
alt: entferne die Subdomains bis du den gemeinsamen Domainteil gefunden hast, in diesem Beispiel <i>example.com</i>.
neu: entferne die Subdomains, bis du den gemeinsamen Domainteil gefunden hast; in diesem Beispiel <i>example.com</i>.
Evtl. auch "Anteil der Domain"

alt: Nun füge einen Punkt vor den gemeinsamen Domain-Namen und füge diesen in das Feld ein.
neu: Nun füge einen Punkt vor den gemeinsamen Domain-Namen und trage diesen in das Feld ein.